install.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+set -e
+
+if [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-apple-darwin"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-unknown-linux-musl"
+elif [ "$(uname -s)" = "Linux" ] && ( uname -m | grep -q -e '^arm' -e '^aarch' ); then
+    target="arm-unknown-linux-gnueabihf"
+else
+    echo "Unsupported OS or architecture"
+    exit 1
+fi
+
+fetch()
+{
+    if which curl > /dev/null; then
+        if [ "$#" -eq 2 ]; then curl -SL -o "$1" "$2"; else curl -sSL "$1"; fi
```

Comment:
Nice solution!
`-S` only has an effect if `-s` is used, so:
```suggestion
        if [ "$#" -eq 2 ]; then curl -L -o "$1" "$2"; else curl -sSL "$1"; fi
```